### PR TITLE
proxy fix

### PR DESCRIPTION
**Pull request by yunstanford**
request from https://github.com/channelcat/sanic/issues/801

### CODE REVIEW

File: sanic/request.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def raw_args(self):
     @property
     def cookies(self):
         if self._cookies is None:
-            cookie = self.headers.get('Cookie') or self.headers.get('cookie')
+            cookie = self.headers.get('Cookie')
```
Comment (r0fls) [inline sanic/request.py]: Is this needed? What does this have to do with the remote address change?

Comment (yunstanford) [inline sanic/request.py]: it's not related. this is code cleanup, since `header` is a Case Insensitive Dict.

**Comment by andreymal** [comment]
I have a small question: is it really necessary to return an empty string when `X-Forwarded-For` is not available? IMHO `self._remote_addr = self.ip` would be more convenient

**Comment by ahopkins** [comment]
@andreymal Can you either open a new issue, or put the question on [the forums](https://community.sanicframework.org) with context relating to current? This is a pretty old PR.

**Comment by andreymal** [comment]
@ahopkins discussion of this PR in a place other than this PR doesn't seem like a good idea ¯\\(°_o)/¯

Also, your link is not working (502 Bad Gateway)

**Comment by ahopkins** [comment]
@andreymal Sorry for the `502`, I was updating the discourse install just now.

There really is no discussion to be had on this PR. It is merged two years ago, and itself does not have a lengthy conversation. If there is something in this PR that you think needs to be changed, it warrants its own conversation.
